package.json

```diff
@@ -168,22 +176,22 @@
     "eslint-plugin-cypress": "^2.12.1",
     "eslint-plugin-import": "2.25.4",
     "eslint-plugin-prettier": "^4.0.0",
-    "husky": "^7.0.4",
-    "jest": "^27.5.1",
-    "jest-html-reporter": "^3.4.2",
-    "jest-junit": "^13.2.0",
+    "husky": "^8.0.1",
+    "jest": "^29.1.2",
+    "jest-html-reporter": "^3.7.0",
+    "jest-junit": "^14.0.1",
     "jsonwebtoken": "^8.5.1",
     "lint-staged": "^11.2.6",
-    "mocha": "^9.2.2",
-    "mocha-junit-reporter": "^2.0.2",
+    "mocha": "^10.0.0",
+    "mocha-junit-reporter": "^2.1.0",
     "nock": "^13.2.4",
     "nodemon": "^2.0.19",
     "path-parse": "^1.0.7",
     "pdf-parse": "^1.1.1",
     "prettier": "^2.7.1",
     "redis-mock": "^0.56.3",
     "sass": "^1.50.1",
-    "ts-jest": "^27.1.4",
-    "typescript": "^4.8.3"
+    "ts-jest": "^29.0.3",
+    "typescript": "^4.8.4"
```

Comment:
updated a couple of other dev dependencies. I also tried to update eslint but there's a few too many code issues, so chickened out.